src/formatDate.js

```diff
@@ -9,6 +9,40 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const dateObj = {};
+  const splitDate = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
The delimiter used for splitting the date string is assumed to be the last element of the `fromFormat` array. Ensure that this assumption is valid in all cases, as it may not always hold true. Consider validating or explicitly defining the delimiter.